pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -266,6 +266,22 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 				v1.EventTypeWarning,
 				string(stork_api.ApplicationBackupStatusFailed),
 				message)
+
+			// Cancel any backups that might have started and mark the backup as failed
+			err := a.Driver.CancelBackup(backup)
+			if err != nil {
+				message := fmt.Sprintf("Error cancelling ApplicationBackup for volumes: %v", err)
+				log.ApplicationBackupLog(backup).Errorf(message)
```

Comment:
if driver is unable to cancel backup do we need to retry here to cancel backup and then mark backup as failed

Reply:
Added exponential backoff for cancel